base/Dockerfile

```
@@ -1,11 +1,14 @@
-FROM continuumio/miniconda3:4.8.2
+FROM continuumio/miniconda3:latest
```

Nitpick: I'm inclined to pin this image to a specific tag to avoid unexpected breakages. Thoughts? 

```suggestion
FROM continuumio/miniconda3:4.9.2
```

---

My worry is that pinning doesn't seem to get updated often. This means we will likely not be picking up any security updates for the underlying OS.

My preference would be to always build the latest Dask image against the latest upstream image. If our users have stability concerns they can pin to a specific version of our image. I don't feel it's appropriate for us to do this on their behalf.

Perhaps we should consider adding some tests here to increase confidence.

---

I'd even make this version a [build-time variable](https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg) with a sensible default.

---

Sounds good 👍